playwright.config.ts

```diff
@@ -27,7 +29,7 @@ export default defineConfig({
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: 'https://staging.drive.internxt.com/login',
+    baseURL: 'https://drive.internxt.com/login',
```

Comment:
You need to run the test with the current branch in a local environment, not pointing to the production frontend, the backend could be the production one but not the frontend. 
If you point to the production frontend you are not testing the actual changes.